### DISCUSSION TITLE
Podman vs Docker bridge locking mechanism for multi-core hw.

### DISCUSSION THREAD

**Discussion by lukasmrtvy** [containers/netavark#804] [General]
Is Podman bridge networking affected by the same locking mechanism as Docker is ( https://github.com/moby/moby/issues/42096, https://github.com/moby/moby/pull/44887 ) ?
Thanks

**Comment by Luap99** [comment]
Yes, all network operations share one global lock.
It is not safe to edit network interfaces and firewall rules concurrently without introducing many race conditions where things can go wrong. I am sure there is room for improvements but getting this right is extremely complicated and hard to tests.
